operations/research-and-development/product/product-management-team-handbook.md

```diff
@@ -398,59 +398,59 @@ Activities:
   * In-Code: if \(this.props.buildEnterpriseReady && this.props.isLicensed
   * Review with PM team and get approval from other leads as required
 
-### Feature Development & The Release Process
+### Feature Development and The Release Process
 
 #### Triage
 
 * Participate in 2x weekly Triage meetings \(may be daily towards end of release\)
```

Comment:
These are currently daily meetings. However, not mandatory for all PMs to join every day.